.github/workflows/test.yml

```
@@ -32,9 +32,89 @@ jobs:
         run: pnpm install --frozen-lockfile
       - run: pnpm build
       - run: pnpm test:ci
+      - name: Extract platform
+        if: always()
+        run: echo "platform=$(echo ${{ matrix.os }} | cut -d '-' -f 1)" >> $GITHUB_ENV
+      - name: Upload allure results
+        id: upload-allure-results
+        if: ${{ env.platform == 'ubuntu' && always() }}
```

The `always()` will be always `true`, so we don't need to specify it.

```suggestion
        if: ${{ env.platform == 'ubuntu' }}
```

---

When we specify `always()` here, the step `Upload allure results` still runs even the step `pnpm test:ci` failed.
So that we can generate the failed test cases in the report.

You can check the behavior of `always()` at this repo: https://github.com/hung-cybo/cli-kintone/actions/runs/8534189157

Settings:
- Both `unit` and `e2e` will be failed.
- For `unit`, I don't specify `always()`
- For `e2e`, I specify `always()`

Result:
- For `unit`, It does not generate the report at all.
- For `e2e`, It generates the report for failed and success test cases.

---

@hung-cybo 
Hmm, I understand what you intended!

How about using `!cancelled()`?
It's more accurate for what we want to do.
(We might not want to generate a report when we cancelled tests)

---

@tasshi-me 

> How about using !cancelled()?
Great. It works as expected.
So, we will use `!cancelled()` instead of `always()`